internal.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+# This script is used by crc developer or internal CI to build the KAO/KCMO patch images to
```

Comment:
`This script is used to build patched KAO/KCMO images with 1 year certificates, and then to push them to quay.io/crcont. The provided pull secret should allow push access to `quay.io/crcont`.